Lib/fontTools/ttLib/sfnt.py

```diff
@@ -556,6 +556,7 @@ def calcChecksum(data):
 		3655064932
 	"""
 	remainder = len(data) % 4
+	data = data[:]          # py3 data is a bytearray so copy it before modifying
```

Comment:
no, data is an immutable `bytes` object, it's not a mutable `bytearray` as the comment incorrectly is saying

Reply:
I agree. I'll take this out. I was passing a bytearray from another program. So the bug is in the call site not here. Sorry about that.